CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
+- Add ready only user profile in the learner dashboard preferences page.
```

Comment:
> Why not simply add a "Profile" entry in the sidebar than automatically redirect to OpenEDX instead?

We want the user to be able to check it's informations without changing website. Ideally, as it was discuss on standup, we would like to make this form editable from our dashboard.

> users will try to click on every inputs [...] then resulting in much frustration.

We may have a issue with [Cunningham](https://github.com/openfun/cunningham) disabled input there.

> ...those fields are read only, inputs should not be used here

May be, it's a choice i made as:
* it's a common way to display theses informations. As for the user as for the dev, we're used to see and code theses interfaces.
* it may become editable.
* it's look like our address form were we've a button that open the creation form.

Reply:
Like discussed during daily last week, later user will be able to update its information in this view so as a first step, read only profile, I think it's good idea to use disabled inputs.